lib/linters/redirect_back_linter.rb

```diff
@@ -0,0 +1,51 @@
+module RuboCop
+  module Cop
+    module IdentityIdp
+      # This lint ensures `redirect_back` is called with
+      # the fallback_location option and allow_other_host set to false.
+      # This is to prevent open redirects via the Referer header.
+      #
+      # @example
+      #   #bad
+      #   redirect_back
+      #   redirect_back '/'
+      #   redirect_back allow_other_host: false
+      #
+      #   #good
+      #   redirect_back fallback_location: '/', allow_other_host: false
+      #
+      class RedirectBackLinter < RuboCop::Cop::Cop
```

Comment:
another linter we probably don't need, especially since this is a headless app for now